setup.cfg

```diff
@@ -1,38 +1,6 @@
-[tool:pytest]
-addopts =
-    --doctest-modules
-    --ignore=esmvalcore/cmor/tables/
-    --cov=esmvalcore
-    --cov-report=xml:test-reports/coverage.xml
-    --cov-report=html:test-reports/coverage_html
-    --html=test-reports/report.html
-env =
-    MPLBACKEND = Agg
-log_level = WARNING
-markers =
-    installation: Test requires installation of dependencies
-    use_sample_data: Run functional tests using real data
-
-[coverage:run]
-parallel = true
-[coverage:report]
-exclude_lines =
-    pragma: no cover
-    if __name__ == .__main__.:
-    if TYPE_CHECKING:
-
 [pycodestyle]
```

Comment:
love the outcome of this issue, that was proposing support for `pyproject.toml` :rofl: https://github.com/PyCQA/pycodestyle/issues/813

Reply:
We should be able to drop pycodestyle as soon as we're using ruff for linting too.